packages/cli/lang/en.lyaml

```diff
@@ -837,6 +837,22 @@ en:
           disableInitial: "Passing the \"--disable-initial\" option is no longer necessary. Running \"hs watch\" no longer uploads the watched directory by default."
           initialUpload: "To upload the directory run \"hs upload\" beforehand or add the \"--initial-upload\" option when running \"hs watch\"."
           notUploaded: "The \"hs watch\" command no longer uploads the watched directory when started. The directory \"{{ path }}\" was not uploaded."
+      preview:
+        describe: "Watch and upload a directory on your computer for changes and start a local development server to preview files"
```

Comment:
Maybe 
`Watch and upload a directory on your computer for changes`
would be better as:
`Upload a directory on your computer and watch for changes`

Reply:
Updated